### PR TITLE
Update KEB service description docs

### PR DESCRIPTION
**Pull request by ksputo**
**Description**

Changes proposed in this pull request:

- Updated available plans with parameters in KEB's service description

**Related issue(s)**

### CODE REVIEW

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 1]
```diff
@@ -69,7 +91,65 @@ These are the provisioning parameters for Azure that you can configure:
...
+ </details>
+
+  <details>
+  <summary label="azure-lite-plan">
+  Azure HA
+  </summary>
+
+| Parameter name | Type | Description | Required | Default value |
+| ---------------|-------|-------------|:----------:|---------------|
+| **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D4_v3` |
```
Comment (PK85) [inline docs/kyma-environment-broker/03-01-service-description.md:104]: ?! there is no such machine type technical name in AWS, machineTypes: []string{"m5.2xlarge", "m5.4xlarge", "m5.8xlarge", "m5.12xlarge", "m4.2xlarge", "m4.4xlarge", "m4.10xlarge", "m4.16xlarge"},

Comment (ksputo) [reply docs/kyma-environment-broker/03-01-service-description.md:104]: copy-paste typo :(

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 2]
```diff
@@ -121,14 +223,19 @@ These are the provisioning parameters for the Trial plan that you can configure:
...
 ```
-The mapping between the platform region and the provider region (Azure or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
+The mapping between the platform region and the provider region (Azure, AWS or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
 
  </details>
  </div>
 
+## Free plan
+
+Free plan allows you to install Kyma on Azure or AWS. The configuration and assumptions are the same as for the Trial plan.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:240]:
```suggestion
Free plan allows you to install Kyma on Azure or AWS. The configuration and assumptions are the same as for the trial plan.
```

Comment (klaudiagrz) [reply docs/kyma-environment-broker/03-01-service-description.md:240]: Are you sure about this sentence? AFAIK, the free plan is not uninstalled after 30 days. Let's confirm.

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 3]
```diff
@@ -121,14 +222,18 @@ These are the provisioning parameters for the Trial plan that you can configure:
...
 ```
-The mapping between the platform region and the provider region (Azure or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
+The mapping between the platform region and the provider region (Azure, AWS or GCP) is defined in the configuration file in the **APP_TRIAL_REGION_MAPPING_FILE_PATH** environment variable. If the platform region is not defined, the default value is `europe`.
 
  </details>
  </div>
 
+## Free plan
+
+Free plan allows you to install Kyma on Azure or AWS. The configuration is the same as for the trial plan, but Kyma is not uninstalled after 30 days.
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:239]:
```suggestion
Free plan allows you to install Kyma on Azure or AWS. The configurable provisioning parameters are the same as for the trial plan.
```

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**